README.md

```diff
@@ -523,11 +523,11 @@ E.g.:
 Request
 
 ```
-curl -L https://gratipay.com/username/payment-instructions.json \
```

Comment:
The heading of this section should be changed too ( **/`%username`/payment-instructions.json**)